easy_translate.gemspec

```diff
@@ -3,6 +3,7 @@ require File.expand_path('lib/easy_translate/version', File.dirname(__FILE__))
 spec = Gem::Specification.new do |s|
   s.name = 'easy_translate'
   s.author = 'John Crepezzi'
+  s.add_development_dependency('google-cloud-translate-v3')
```

Comment:
You have to add this as a proper dependency (ie. `add_dependency` instead of `add_development_dependency`) - that's probably the reason require fails.

Reply:
Thanks, it works now.